pkgs/top-level/all-packages.nix

```diff
@@ -13605,13 +13605,13 @@ in
 
   flite = callPackage ../development/libraries/flite { };
 
-  fltk13 = callPackage ../development/libraries/fltk {
+  inherit (callPackage ../development/libraries/fltk {
```

Comment:
Using `inherit (callPackage ...` prevents `fltk.override` from working. Consider moving `callPackage` into the definition of `generic`.

Reply:
I couldn't figure out another way that works. If you have an example of another package that does this, it would be much-appreciated.

Reply:
There are ways to do that. A common convention is to have

* pkgs/development/libraries/fltk/1.3.nix
* pkgs/development/libraries/fltk/1.4.nix
* pkgs/development/libraries/fltk/generic.nix

Reply:
Thanks for the hint, it's done.